### ISSUE TITLE
Wrangling to cellxgene

### ISSUE THREAD

**Issue by Wkt8** [ebi-ait/dcp-ingest-central#341]
There is a task to explore the possibilities of providing a standardised form of CGMs into CZI’s CellxGene platform (https://cellxgene.cziscience.com/). 

This ticket is simply to try to take a single project's CGMs, and 
- figure out what format CellxGene accepts
- how to edit the CGM into that format, and how much effort this will roughly take 
- how much effort curating to extract cell type annotation and add standard ontologies would take
- potential methods of streamlining this process 

Useful Information:
- The CellxGene schema: https://github.com/chanzuckerberg/single-cell-curation/blob/ambrosecarr/schema-v1.1.1/schema/1.1.1/corpora_schema.md
	- CellxGene detailed format requirements: https://github.com/chanzuckerberg/single-cell-curation/blob/ambrosecarr/schema-v1.1.1/schema/1.1.1/anndata_encoding.md
	- Validation tools we could use to check CellxGene inputs: https://github.com/chanzuckerberg/single-cell-curation

**Comment by amnonkhen** [comment]
@Wkt8 would this be considered another archiving target (as BioSamples, BioStudies, ENA and SCEA)?

**Comment by Wkt8** [comment]
Yes - we have some tools from the UCSC wranglers that will help with wrangling to cellxgene as another archiving target

**Comment by Wkt8** [comment]
HHave had some useful and fruitful meetings - minutes found here. 

Targets for January are: 
- Ambrose Happy to give us some cellxgene copy to get faster replies to emailing contributors
- I made it clear to Ambrose the pain points (getting raw matrices, ENSEMBL gene IDs, and particularly that we were removing certain genes not in ENSEMBL from matrices)
-Don't care about DCP-generated matrices
   -standardize on a specific nomenclature (X.DCP-gen) etc.
   -2 issues about matching up HCA and cellxgene schema:

Cell lines
- organoids
- how when, timelines

Actions on EBI:
How many terms have we used from HsapDv vs. EFO vs. UBERON for projects in the DCP?
How do we want to modify the metadata schema to have a field for type of biomaterial, e.g. (cell line, tissue, etc.)
